content/en/docs/tasks/traffic-management/egress/egress-gateway-tls-origination-sds/index.md

```diff
@@ -322,7 +321,7 @@ to hold the configuration of the NGINX server:
             number: 443
           tls:
             mode: SIMPLE
-            credentialName: client-credential # this must match the secret created earlier without the "-cacert" suffix
+            credentialName: client-credential # this must match the secret created earlier, if the secret name ends with "-cacert", "-cacert" suffix can be ignored
```

Comment:
The "secret created earlier" DOES have a `-cacert` suffix. There is no "if" in this case.

Reply:
Here, I want to express that if the secret name is not suffixed with "-cacert", the credentialName must be exactly matched with the secret name, if so, then "-cacert" suffix can be ignored. do you suggest I change the description to " because the secret name ends with "-cacert", "-cacert" can be ignored here" ?

Reply:
I would not try to explain that in the DestinationRule comment.

Seems to me that there are some strange rules for secret names ending in `-cacert` that should be documented, probably in here: https://preliminary.istio.io/latest/docs/reference/config/networking/destination-rule/#ClientTLSSettings.

The earlier step in this task said that `the secret name for an Istio CA-only certificate must end with -cacert`, but you are also changing that. If that's no longer the case, then why not change the secret name in this task to not have the `-cacert` suffix. That way the task doesn't even need to address the issue at all.

cc @istio/wg-security-maintainers

Reply:
https://istio.io/latest/docs/tasks/traffic-management/ingress/secure-ingress/#configure-a-mutual-tls-ingress-gateway (bottom of that section, I think we are missing a header...) explains this.

Basically, there are various formats possible:
```
A TLS Secret with keys tls.key and tls.crt, as described above. For mutual TLS, a ca.crt key can be used.
A generic Secret with keys key and cert. For mutual TLS, a cacert key can be used.
A generic Secret with keys key and cert. For mutual TLS, a separate generic Secret named <secret>-cacert, with a cacert key. For example, httpbin-credential has key and cert, and httpbin-credential-cacert has cacert.
```